music-playlist-app/index.js

```diff
@@ -83,8 +86,19 @@ function displaySong(song = "", artist = "") {
 }
 
 function playPauseSong() {
-  isPlaying ? song.pause() : song.play();
-
+  if (isPlaying) {
+    song.pause();
+    playPauseBtn.setAttribute(
+      "aria-label",
+      `Paused ${playlist[currentSong].title}`
+    );
+  } else {
+    song.play();
+    playPauseBtn.setAttribute(
+      "aria-label",
+      `Playing ${playlist[currentSong].title}`
```

Comment:
```suggestion
      `Pause ${playlist[currentSong].title}`
```

Same logic as above, when the song is playing then the button should read "Pause [song name]" since that's what will happen when you click on the button.